### PR TITLE
Feat/crash tracking 

### PR DESCRIPTION
**Pull request by rhakafebriandi**
### Background

Implement Crashlytics to our mobile apps for every current implemented try-catch situation & API calls
### Impacted Products

all screen process

## firebase Crashlytics dashboard
image

### How to Test

Please include steps of how you tested this changes

### Pre-Deployment Checklist

- [x] Run Local OK (Mandatory)
- [x] Run Dev Build on Real Device

### Test Checklist

- [ ] Unit Testing
- [ ] Other (Please Elaborate)

### Code Review Guidelines

https://oyteam.quip.com/4m8kAafMWWiX/Code-Review-Manifesto

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -43,38 +43,37 @@ dependencies:
...
+  firebase_dynamic_links: ^5.0.17
   cached_network_image: ^3.2.0
-  firebase_remote_config: ^2.0.1
+  firebase_remote_config: ^3.0.15
   shimmer: ^2.0.0
   version: ^3.0.2
   store_redirect: ^2.0.1
   flutter_svg: ^1.0.3
-  audio_service: ^0.18.9
+  firebase_crashlytics: ^3.0.17
```
Comment (indraseptama) [inline pubspec.yaml:73]: di sini kenapa ada beberapa versi yang naik ya rhak? atau karena firebase nya dinaikin kah? Ada penjelasannya ga ya? @rhakafebriandi 

takutnya ngebreak flow yang sekarang cc @yumnapratista24

Comment (rhakafebriandi) [reply pubspec.yaml:73]: ittu harusnya cuma untuk library firebase nya aja kak, aku coba ganti dulu deh buat itu

**Comment by yumnapratista24** [comment]
Mau nanya juga, ini berarti baru nge init aja ya ? klo kejadian error di try catch blm include ya brrt ?

**Review by indraseptama — APPROVED** [review]
lgtm!

**Review by yumnapratista24 — APPROVED** [review]

**Pull request merged**